### ISSUE TITLE
 <Plug>VimspectorBalloonEval does not work in VimR

### ISSUE THREAD

**Issue by qianwch** [puremourning/vimspector#604]
### Description

I have checked the code, it seems that the following code triggers the bug:
```
    self._is_term = not bool( int( vim.eval( "has( 'gui_running' )" ) ) )
``` 

It works fine with nvim terminal and nvim-qt. 

### Works in vim?

Only broken in Neovim

### Works with clean config?

Yes

### Minimal Reproduction

List of steps to reproduce:

1. Open VimR and open a java project
2. Set breakpoint
3. Launch debugger.
4. use <Plug>VimspectorBalloonEval to evaluate variables

### Expected Behaviour

Variable ballon popup normally.

### Actual Behaviour

![image](https://user-images.githubusercontent.com/45196316/174703053-2ff55228-c1e7-4d9b-9597-a60df1502426.png)

### Installation Type

Plugin Manager (git clone)

### Vimspector version

b33b1be7679efb756c88ad398f6e7d77505c02c6

### Vim version

```Text
0.7.0
```

### Python version

3.9.13

### Operating System

Mac OS 12.4

### Declaration

- [X] I have read and understood [CONTRIBUTING.md](https://github.com/puremourning/vimspector/blob/master/CONTRIBUTING.md#diagnostics)
- [X] I have read and understood the [CODE_OF_CONDUCT.md](https://github.com/puremourning/vimspector/blob/master/CODE_OF_CONDUCT.md)

**Comment by puremourning** [comment]
I'll check later but if we are passing invalid args to that function, it's worth fixing it. Surprising it doesn't happen in other places.

**Comment by puremourning** [comment]
Fixed by #605
